### DISCUSSION TITLE
11/9(土)20:00~  Backend ミーティング #2

### DISCUSSION THREAD

**Discussion by ayase-mstk** [42-pong/42-pong#17] [Meetings]
議題
- 進捗共有
- アーキテクチャについて
- /healthエンドポイントの実装タスク
- 時間あれば次のフェーズのこと
- 次回タスク振り分け

**Comment by habvi** [comment]
進捗
- PR #19 
- PR #21 

backlog に追加した細かめ task
- `compose.yaml` : healthcheck 追加
- `backend/Dockerfile` : UESR 指定
- `backend/tools/entrypoint.sh` : Django の初期設定 (superuser作成, timezone変更)
- `compose.yaml` と `backend/Dockerfile` で `WORKDIR` を同期する

**Comment by habvi** [comment]
議事録
- openapi 
  - 一旦 servers の url : `localhost:8000` or `localhost:8000/api`
- オニオンアーキテクチャにする #25 
- dev branch のルールについて相談
  main <- dev <- feature-xxx
  main <- dev は branch 切って phase 名で PR 作る (wiki の branch rule 参照)

---

次のタスク
- /health get 実装
- DB を postgresql にする
- swagger-ui が見れるようにする
- PR template 追加

---

次のフェーズ
- onion 風の実装ができる endpoint 作る？ -> user 作成時でも良いかも
